multiqc/modules/samtools/stats.py

```diff
@@ -153,6 +153,18 @@ def alignment_section(self, samples_data):
             name = 'Percent Mapped',
             anchor = 'samtools-stats-alignment',
             description = "Alignment metrics from <code>samtools stats</code>; mapped vs. unmapped reads.",
+            helptext="""For a set of samples that have come from the same multiplexed library, 
+        similar numbers of reads for each sample are expected. Large differences in numbers might 
+        indicate issues during the library preparation process. Whilst large differences in read 
+        numbers may be controlled for in downstream processings (e.g. read count normalisation), 
+        you may wish to consider whether the read depths achieved have fallen below recommended 
+        levels depending on the applications (e.g. ~ 10 million reads for differential gene
+        expression analysis.)</p>
+        <p>A high-quality library should have ~95% of reads align. Low alignment rates could
```

Comment:
I think this sentence needs a little bit more qualification as it depends so heavily on the library type. If this is used with a bisulfite mapper, 80% alignment is a great result. If it's non-Human then 50% can be a great result etc. etc.

Reply:
I removed all specific stuff from there for now.